tools/legacy/sample_vpp/Android.mk

```diff
@@ -0,0 +1,60 @@
+# Copyright(c) 2021 Intel Corporation
```

Comment:
Can we change the Copyright time to 2023 here?

Thanks,
Tianmi

Reply:
updated